src/shared/quota-util.c

```diff
@@ -40,3 +43,45 @@ int quotactl_path(int cmd, const char *path, int id, void *addr) {
 
         return quotactl_devnum(cmd, devno, id, addr);
 }
+
+int quota_proj_id_is_supported(int fd, uint32_t proj_id, struct dqblk *req) {
```

Comment:
so i find the function name a bit weird, because it's not just checking if the feature is supported, it actively reads the quota data, and returns it.

maybe call it quota_query_proj_id() or so?